v23.1/architecture/replication-layer.md

```diff
@@ -207,6 +209,23 @@ Rebalancing is achieved by using a snapshot of a replica from the leaseholder, a
 
 In addition to the rebalancing that occurs when nodes join or leave a cluster, replicas are also rebalanced automatically based on the relative load across the nodes within a cluster. For more information, see the `kv.allocator.load_based_rebalancing` and `kv.allocator.qps_rebalance_threshold` [cluster settings](../cluster-settings.html). Note that depending on the needs of your deployment, you can exercise additional control over the location of leases and replicas by [configuring replication zones](../configure-replication-zones.html).
 
+### Important values and timeouts
+
+The following table lists some important values used by CockroachDB's replication layer:
+
+Constant | Default value | Notes
+---------|---------------|------
+[Raft](#raft) election timeout | {{site.data.constants.cockroach_raft_election_timeout_ticks}} * {{site.data.constants.cockroach_tick_interval}} | Controlled by `COCKROACH_RAFT_ELECTION_TIMEOUT_TICKS`, which is then multiplied by the default tick interval to determine the timeout value. This value is then multiplied by a random factor to avoid election ties.
+[Raft](#raft) proposal timeout | {{site.data.constants.cockroach_raft_reproposal_timeout_ticks}} * {{site.data.constants.cockroach_tick_interval}} | Controlled by `COCKROACH_RAFT_REPROPOSAL_TIMEOUT_TICKS`, which is then multiplied by the default tick interval to determine the value. This value is then multiplied by a random factor to avoid election ties.
```

Comment:
I'd remove the last sentence. This doesn't really have anything to do with elections, it's how often we repropose/retry a Raft command if we haven't gotten a result yet. It can technically happen within a factor of 1-2 of this, because we only check it once per reproposal interval, but I don't think we need to get that technical.